src/styles/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  display: flex;
+  margin: 0;
+  &__list {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 15px;
+    text-transform: uppercase;
+  }
+
+  &__item {
+    list-style: none;
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    display: block;
+    color: $dark-blue-color;
+    text-decoration: none;
+    padding: 22.5px 0;
```

Comment:
#4 in checklist
[STYLES] - If you need to adjust positioning with 1px or use fractional numbers like 0.5 ANYWHERE, you are doing something wrong, change the alignment technique you use

Reply:
agree